lib/amazon_business_api/client.rb

```diff
@@ -79,10 +79,17 @@ def request_access_token # rubocop:disable Metrics/MethodLength
       data
     end
 
+    def headers
+      {
+        'x-amz-access-token' => access_token,
+        'x-amz-user-email' => ENV['AMAZON_BUSINESS_API_EMAIL']
```

Comment:
We don't need line 85 everytime. I had to add it for OrderingAPI calls. Quick patch.

Reply:
we have `email` available on our client